htdocs/core/lib/pdf.lib.php

```diff
@@ -1502,7 +1502,7 @@ function pdf_getlinedesc($object, $i, $outputlangs, $hideref = 0, $hidedesc = 0,
 			} else {
 				$textwasnotmodified = ($desc == $prodser->description);
 			}
-			if (!empty($prodser->multilangs[$outputlangs->defaultlang]["description"])) {
+			if (!empty($prodser->multilangs[$outputlangs->defaultlang]["label"])) {
```

Comment:
If you have a product with no description in other language, the pdf will display the default language description, find image below

![BugTraductionDescription](https://github.com/Dolibarr/dolibarr/assets/46494485/c3df9217-432b-4e4b-b84c-8bae4d26e212)

Reply:
Is your screen shot result before the change (so the trouble) or after (so the fix) ?

PS: You can paste your image directly into the textarea of github to publish an image instead of building an page with n image to export it after. You should save time.

Reply:
Yes before the change:
![image](https://github.com/Dolibarr/dolibarr/assets/46494485/d02df9c5-f17f-4d9f-af62-d03a3f0100ba)

and after the change:
![image](https://github.com/Dolibarr/dolibarr/assets/46494485/0134fccb-fb32-4ad9-92bf-0c3c437c34aa)